pkg/models/plan_collaborator.go

```diff
@@ -7,6 +7,7 @@ type PlanCollaborator struct {
 	EUAUserID string   `json:"euaUserId" db:"eua_user_id"`
 	FullName  string   `json:"fullName" db:"full_name"`
 	TeamRole  TeamRole `json:"teamRole" db:"team_role"`
+	Email     string   `json:"email" db:"email"`
```

Comment:
Would you mind leaving a comment (either on the struct as a whole, or on the `FullName` and `Email` fields) describing that these are stored as a result from the initial CEDAR query that's made when we create the collaborator (effectively caching these values, which are unlikely to change)?

Reply:
b39e78894448f43c13dfa5e6f99d71763100b7b7